keyboards/crkbd/keymaps/jpe230/rules.mk

```diff
@@ -1,12 +1,25 @@
 # Shared rules for both of my Corne
 OLED_ENABLE       = yes
 OLED_DRIVER       = SSD1306
-LTO_ENABLE        = yes
 VIA_ENABLE        = yes
+RGB_MATRIX_ENABLE = yes
 
-# Rules for my Corne without RGB
-# WPM_ENABLE        = yes
-# MOUSEKEY_ENABLE   = yes
+SRC += oled/oled_handler.c process_record.c
+
+ifneq ($(CONVERT_TO),)
```

Comment:
Couldd and probably should move this to `post_rules.mk` and then you could check MCU or BOARD, instead, I think.

Reply:
Thanks for the suggestion Drashna!

That's exactly the first thing I tried! Sadly, I believe that there is not support for `post_rules.mk` for user keymaps...

Reply:
Aah, it might not be, yeah.